bots/pr/src/main/java/org/openjdk/skara/bots/pr/BackportCommand.java

```diff
@@ -115,10 +200,33 @@ public void handle(PullRequestBot bot, HostedCommit commit, LimitedCensusInstanc
         }
         var targetBranch = new Branch(targetBranchName);
 
+        // Find real user when the command user is bot
+        HostUser realUser = command.user();
+        if (realUser.equals(bot.repo().forge().currentUser())) {
+            var botComment = allComments.stream()
+                    .filter(comment -> comment.author().equals(bot.repo().forge().currentUser()))
+                    .filter(comment -> comment.body().contains("Backport for repo `" + repoName + "` on branch `" + targetBranchName + "` was successfully enabled."))
```

Comment:
Instead of parsing the human readable message, I would recommend encoding a more structured data entry in an html comment. That will be easier to parse and less sensitive to future changes of this code. There should be several similar examples of that throughout the bots.

Reply:
Sure. Will fix it

Reply:
I don't think you changed this.

Reply:
Sorry, I misunderstood your advice. I will fix it again.